### ISSUE TITLE
Extra PDF files

### ISSUE THREAD

**Issue by gnott** [microsimulation/ijm-xml#98]
I noticed some PDF files named slightly different than the standard PDF file naming for the article PDF. In reviewing some of the XML files for these articles, I do not see any reference to these PDF files. I opened one of these PDFs now, and maybe they are the PDF files from the old IJM journal site?

```
ijm-00107-A.pdf
ijm-00116a.pdf
ijm-00117a.pdf
ijm-00123A.pdf
ijm-00156A.pdf
ijm-00165A.pdf
ijm.00168A.pdf
ijm-00188a.pdf
ijm-00190a.pdf
```

@Vijayarajmurugesan could you please look in the zip files for the associated articles and remove these PDF files if they are not to be part of the final content package? Thanks!

**Comment by Vijayarajmurugesan** [comment]
@gnott Thanks for checking with us. If the articles contain an appendix section it has separate PDF provided in the input materials (appendix text). In the XML files, we delivered as a single combined file of main text and appendix. If you need the single pdf file, we need to merge with the main text pdf and appendix pdf. Could you suggest and advise, how to proceed this?

**Comment by gnott** [comment]
I didn't look at the PDF files themselves, and I understand their presence is intentional and they are not main articles but appendices - thanks for the clarification.

What I looked for was any mention of the particular PDF files in the XML, that would link it some how to the article, so it could be displayed or associated with the article. Without that, or with a manifest to indicate the article assets, it would be difficult to determine how to make these online appendix PDFs available to the reader.

The issue is beyond my realm of decision making, and I will bat this issue back to @Melissa37 please for comment.

**Comment by Melissa37** [comment]
Hi both

This is interesting. If stitching the PDFs together into one PDF is not achievable, then we will have to think of another solution for when we move to Libero.
For the test site, I suggest that we understand the appendix PDFs will not be linked anywhere.

@gnott You can still get the articles to display on Continuum?

@thewilkybarkid and @matteorichiardi FYI

**Comment by gnott** [comment]
These PDF files being omitted from the demo site should work with no error. My question was more related to Libero and how to obtain the best content zip package + XML combination.

**Comment by matteorichiardi** [comment]
Hi both
I think we already discussed this and decided to add the appendix at the
end of the paper in the html page. As for the pdf, we can simply upload the
standard article and remove the  pdf appendix (html only when appendices
are long : this is fine). Thanks,
Matteo
Matteo

**Comment by Vijayarajmurugesan** [comment]
@gnott @Melissa37 
As Matteo said in the above comment, we will remove the appendix pdf and resupply the updated package.
Please let me know this is fine with you
Regards,
Vijay

**Comment by Melissa37** [comment]
Great, thanks!

**Comment by Vijayarajmurugesan** [comment]
@Melissa37 We have removed that all extra appendix pdfs in the package folder. We have done this one by June month itself. Could you please let me know if you need to be an updated package folder.
Thanks,
Vijay

**Comment by Melissa37** [comment]
@Vijayarajmurugesan can you confirm the content of the PDFs is in the XML now?

**Comment by Vijayarajmurugesan** [comment]
@Melissa37 Yes, I confirm that all the content of the PDFs is in the XML only.
